share/governance.md

```diff
@@ -0,0 +1,80 @@
+
+* advocating for use of the project
+* informing developers of project strengths and weaknesses from a new user’s perspective
+* providing moral support (a ‘thank you’ goes a long way)
+* writing documentation and tutorials
+* filing bug reports and feature requests
+* participating on the discussion board
+
+
+Users who continue to engage with the project and its community will often find themselves becoming more and more involved. Such users may then go on to become contributors, as described below.
+
+*How to become one*: Use a VoxPupuli project
+
+
+Collaborators
+-------------
+
+
+Collaborators are community members who submit patches to the project. These patches may be a one-time occurrence or occur over time. Expectations are that contributors will submit patches that are small at first and will only grow larger once the contributor has built confidence in the quality of their patches.
+
+*How to become one*: Submit a pull request to a VoxPupuli project
+
+
+Contributors
+------------
+
+
+Contributors are collaborators who have shown wide dedication to the VoxPupuli project in general or deep dedication to one project in particular, and the ability to work well with collaborators and users. The contributors have responsibilities beyond the collaborators. In particular, contributors formally decide on whether a pull request is merged or a release is cut. A contributor will use lazy consensus to decide on whether to merge a pull request from a collaborator. If the discussion is no longer moving towards a consensus, the PMC must vote via lazy consensus on whether the patch should be applied.
+
+*How to become one*: Be a collaborator and be nominated to the PMC as a contributor. Nominations should be sent to <email address for nominations>. You may nominate yourself.
+
+
+Project Management Committee (PMC)
+----------------------------------
+
+
+The project management committee has responsibilities beyond contributors that include participating in strategic planning, release planning, and approving changes to the governance model. They also have the duty of upholding the community code of conduct and ensuring its values. The PMC is also responsible for updating the community code of conduct. It also makes decisions when community consensus cannot be reached. 
+
+The PMC has final say over who can become a committer and will use lazy consensus for approval. Discussion over committer nominations will be done in private.
+
+Membership of the PMC is by election. Condorcet voting is held once a year. A non-running election official is appointed by the PMC and runs the election. There are 5 members of the PMC.
+
+*How to become one*: Win in the yearly election.
+
+
+
+Lazy Consensus
```

Comment:
One thing I think is missing is a central low-traffic communication format. Lazy consensus depends on the interested parties being able to passively consume information from the stream. Github Pull requests and #voxpupuli irc chatter are both very noisy and hard to keep up with. Announcements and big proposals should be somewhere that is low-ish traffic and high visibility. Two ideas on what that could be:

1) A twice-monthly irc meeting for an hour
2) A mailing list

We do have a puppet-community google group, but due to the really hard-to-use-if-you-dont-gmail interface of that group, I've never been able to join it, plus it has the wrong name.

The advantage of irc meetings is we can hash things through very quickly sometimes. We also already heavily use irc. The disadvantage is that there is no good time to do it and still have people from Europe, APAC, and the Americas be able to come.

The advantage of a mailing list is that it doesn't disenfranchise people on the far-side of the world from the meeting. It also allows for longer form statements, which are useful when communicating bigger ideas. It also is a much easier format for people who don't speak English as their first language, as they can take time to read slowly and work on their response.

As written I have it using the systems we already use, github pull requests and irc. But just from reading this I can tell that important messages will get lost and it will be hard to have disucssions.